Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -133,19 +139,35 @@ public sealed class BlockStoreLoop
             StartLoop();
         }
 
+        /// <summary>
+        /// Adds a block to Pending Storage
+        /// <para>
+        /// The BlockStoreSignaler calls AddToPending.
+        /// </para>
+        /// <para>
+        /// Only add the block to pending storage if:
+        ///     1: The block does exist on the chain
+        ///     2: The store's tip is less than the block to add's height
+        /// </para>
+        /// </summary>
+        /// <param name="block"></param>
+        /// <remarks>Possibly check the size of pending in memory</remarks>
         public void AddToPending(Block block)
         {
             ChainedBlock chainedBlock = this.Chain.GetBlock(block.GetHash());
             if (chainedBlock == null)
-                return; // reorg
-
-            // check the size of pending in memory
+                return;
 
-            // add to pending blocks
             if (this.StoreTip.Height < chainedBlock.Height)
                 this.PendingStorage.TryAdd(chainedBlock.HashBlock, new BlockPair(block, chainedBlock));
         }
 
+        /// <summary>
+        /// Flush the BlockStore by calling DownloadAndStoreBlocks with disposeMode of true
```

Comment:
I'd go with simple `<summary>Persists the unsaved <see cref="BlockStore"/> to the disk</summary>`. I think there is no need rewrite the code to English.